TESTS/network/multihoming/multihoming_udpsocket_echotest.cpp

```diff
@@ -209,10 +211,12 @@ void MULTIHOMING_UDPSOCKET_ECHOTEST_NONBLOCK()
                     }
                 }
                 loss_ratio = 1 - ((double)udp_stats[j].recv_bytes / (double)udp_stats[j].sent_bytes);
-                printf("Bytes sent: %d, bytes received %d, loss ratio %.2lf\r\n", udp_stats[j].sent_bytes, udp_stats[j].recv_bytes, loss_ratio);
+                tr_info("Bytes sent: %d, bytes received %d, loss ratio %.2lf\r\n", udp_stats[j].sent_bytes, udp_stats[j].recv_bytes, loss_ratio);
                 TEST_ASSERT_DOUBLE_WITHIN(TOLERATED_LOSS_RATIO, EXPECTED_LOSS_RATIO, loss_ratio);
 
 #endif
+            } else {
```

Comment:
And here as well.

Reply:
If no packets were sent then `packets_sent` equals to 0, there is no extra check as in the two cases above?

Reply:
You're right for the multihoming case - this makes sense, 
In the two other cases these lines will not allow the situation where `packets_sent` is 0:
```
        // Make sure that at least one packet of every size was sent.
        TEST_ASSERT_TRUE(packets_sent > packets_sent_prev);
```
If no packets get sent, this assertion fails and we never reach the `if (packets_sent > 0)`, so perhaps we could actually remove this check (although I believe it's left there in case someone modifies the code and forgets the division by 0 situation or for the static analysis?)

Reply:
@michalpasztamobica Fixed the two cases you mentioned, please review.